### ISSUE TITLE
Oh WOW Let's claps for this hero

### ISSUE THREAD

**Issue by gnijuohz** [cirosantilli/china-dictatorship#5]
There is a question in FAQ that I want to talk more about. 

> What is the root of the harm? me or GFW?

The fact you are not the root of the harm doesn't justify the harm you inflict on other human beings. You are basically saying, sure, I'm harming these Chinese programmers, who may or may not even care about politics, but I'm super fine with it since I'm not the root of the problem.

In some twisted self-justifying way, you already don't see Chinese programmers as concrete human beings anymore. That's why you feel okay harming them. You don't just see the Chinese government as the bad guy, you see millions of Chinese programmers as the bad guys.

Are you seeing the Chinese people as the victims of the censorship or the conspirators of the censorship that you are "fighting against"? 

If you see them as the victims, harming them for a cause seems so twisted I can't help finding myself asking, who would do such a thing? Is that so different from "I'm not the cause of the refugee problem, so let's block all their escape routes and watch them die."? After all, what is the root of the harm? Not me!

If you see them as the bad guys as well, that would explains a lot. The whole *programmers fight the dictatorship* idea is so funny I can't help laughing. Believe it or not, these Chinese programmers are so much less politically driven than you are. Most of them are just trying to get their job done for some employer and go back to their family. Some do speak out. Some don't.  Some won't. Some don't even care.

Okay, wait, what did you say your goal is? Getting sites like SO and github banned in China? So?  Then? 

Didn't you yourself say, 

> Chinese programmers all know how to climb the wall.

?

So you try to get the sites banned in China. But you claim "Chinese programmers all know how to climb the wall." (clearly for the purpose of self-justification, since you won't even do this if they ALL know how to claim the wall. Let alone write to explain how you are harming them but that's worth it because your noble ideology is clearly more important than other human beings' lives)

Explain that to me again?

Oh and the price you have to pay! You Chinese programmers, you just don't understand, I'm paying a price for voicing my opinion. Like it's even slightly close to the price that many Chinese people pay by voicing their voices, while physically in China, being a Chinese.

Some just choose to deal with the censorship. Why the heck do you think your ideology is more important than their lives again? Just because you do.

I'm sorry but this so called *fighting for a cause* is such a logically incoherent, self-justifying and twisted act by such an immature, activist wannabe I just want to puke.

No, I don't want you to stop fighting against the censorship. Just do it while not being a jerk to a group of human beings for nothing but your perceived noble cause.

**Comment by cirosantilli** [comment]
Relax bro.

My motivations are not important. Attacking me personally is useless.

All that matters is if this action will work or not.

**Comment by cirosantilli** [comment]
Yes, the price I pay for speaking out is infinitely smaller than the Chinese who speak out, that is why I can do it.

What I meant is that it is comparable to having to use circumvention to access Stack Overflow.

**Comment by gnijuohz** [comment]
I'm not attacking you just like you are not attacking China. I'm speaking out too about you as a Chinese programmer. Kinda like trump can *speak out*, people can also speak out against Trump. I'm sure you understand how important the freedom of speech is.

> my motivations are not important.

I find this funny. Harming people without a clear motivation does not sound like a reasonable thing to do, does it?

And you ask me to relax, bro? 

> it's comparable to having to use circumvention to access Stack Overflow.

Mmmm, seems like you've decided all Chinese programmers need to pay an extra price for their government's censorship, as if they are not already paying.

One motivation that can actually explain everything that you do is simply that you hate all Chinese programmers. To you, they are not real human beings working to make a living and take care of their families, heck, they don't even have feelings like you too, they are just **communist** Chinese programmers working for a **communist** government.

I'm just here to call you out as a jerk. 

> all that matters is if this action will work out or not.

In other words, hurting real people doesn't matter! My heroic activism matters most! Don't question my motivation because I'm fighting the communists! Anything I do is right in the name of fighting the communists!

Relax bro, I'm just here to call you out as a jerk.

**Comment by cirosantilli** [comment]
As mentioned before, I don't think discussing my motivations is meaningful.

You say I'm purely selfish, I say not, and we never gain any new knowledge.

If you have any non obvious new insights on the possible consequences of my actions or what I could do differently I will hear.

:heart:

**Comment by gnijuohz** [comment]
Oh no I'm not calling you selfish. I just question the validity of your so-called *fighting for a cause* and *activism*. It's like we are having a different conversion here, come on, bro!

In summary, I categorize what you are doing as this: trying to harm other human beings in the name of fighting evil. And I just find it ridiculously irrational and laughable.

> I don't think discussing my motivations is meaningful

Translation: I don't think discussing why I am being a jerk to fellow humans is meaningful.

Okay fine. Maybe there is no reason, maybe that's just your kind of thing.

I do have an insight that you seem keep missing: what you do is irrational and twisted from all angles, and somehow you are still doing it. 

Here is the ultimate insight: **you are not a nice human being. You don't have to be unpleasant to fight a cause. You just choose to be unpleasant.** (highlighted in case you missed it)

:no_good:

**Comment by cirosantilli** [comment]
You need to chill out.
